www/front_src/src/Resources/Details/tabs/Timeline/ExportToCsv.tsx

```diff
@@ -1,29 +1,30 @@
 import { useTranslation } from 'react-i18next';
-import { useAtomValue } from 'jotai';
 
-import { Button, Stack } from '@mui/material';
 import SaveIcon from '@mui/icons-material/SaveAlt';
+import { Button, Stack } from '@mui/material';
+
+import { SearchParameter, getSearchQueryParameterValue } from '@centreon/ui';
 
 import { labelExportToCSV } from '../../../translatedLabels';
-import { detailsAtom } from '../../detailsAtoms';
-import {
-  getDatesDerivedAtom,
-  selectedTimePeriodAtom,
-} from '../../../Graph/Performance/TimePeriods/timePeriodAtoms';
 
-const ExportToCsv = (): JSX.Element => {
-  const { t } = useTranslation();
+interface Props {
+  getSearch: () => SearchParameter | undefined;
+  timelineEndpoint: string;
+}
 
-  const details = useAtomValue(detailsAtom);
+const ExportToCsv = ({ getSearch, timelineEndpoint }: Props): JSX.Element => {
+  const { t } = useTranslation();
 
-  const getIntervalDates = useAtomValue(getDatesDerivedAtom);
-  const selectedTimePeriod = useAtomValue(selectedTimePeriodAtom);
+  const timelineDownloadEndpoint = `${timelineEndpoint}/download`;
 
-  const [start, end] = getIntervalDates(selectedTimePeriod);
+  const exportToCsv = (): void => {
+    const data = getSearch();
 
-  const exportToCSVEndpoint = `${details?.links.endpoints.timeline}/download?start_date=${start}&end_date=${end}`;
+    const params = getSearchQueryParameterValue(data);
```

Comment:
```suggestion
    const parameters = getSearchQueryParameterValue(data);
```

No shortcuts in naming